src/PAModel/Utility/Utilities.cs

```diff
@@ -148,16 +148,19 @@ public static JsonElement ToJson(this ZipArchiveEntry e)
         // returns "bar\hi.txt"
 
         // Net Core 2.1 provides a Path.GetRelativePath api
-        // but since we target netstandard, we can convert to URIs and make the relative path from that
+        // but since we target netstandard 2.0, we can convert to URIs and make the relative path from that
         // see https://stackoverflow.com/questions/275689/how-to-get-relative-path-from-absolute-path
-        public static string GetRelativePath(string full, string basePath)
+        // For reference, see Core's impl at: https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/IO/Path.cs#L861
```

Comment:
Perhaps adding Xml documentation to this stating that the first param is always a file and the second param is always a directory might help clarify the expected usage?